persistence/sql/persister_oauth2.go

```diff
@@ -247,6 +292,31 @@ func (p *Persister) findSessionBySignature(ctx context.Context, rawSignature str
 	})
 }
 
+func (p *Persister) findRequesterBySignature(ctx context.Context, signature string, tableName tableName) (fosite.Requester, error) {
```

Comment:
Please add tests for this new method

Reply:
Figured out how to remove this method :)